packages/happo-target-react-native/src/defaultOptions.js

```diff
@@ -0,0 +1,96 @@
+const path = require('path');
+
+const iosAppPath = path.join(
+  __dirname, // path of this file
+  '../', // root of this module
+  'runner/ios/build/Build/Products/Debug-iphonesimulator/HappoRunner.app',
+);
+
+const androidAppPath = path.join(
+  __dirname, // path of this file
+  '../', // root of this module
+  'runner/android/app/build/outputs/apk/app-debug.apk',
+);
+
+// `${process.cwd()}/../android/airbnb/build/outputs/apk/app-debug.apk`;
+
+function fail(message) {
+  /* eslint-disable no-console */
+  console.error('Happo Target React Native config validation failed:');
+  console.error(message);
+  process.exit(1);
+}
+
+
+function validatePassedOptions(options) {
+  if (typeof options.platform !== 'string') {
+    fail(`Expected \`platform\` to be string. Found ${typeof options.platform} instead.`);
+    return false;
+  }
+  switch (options.platform.toLowerCase()) {
+    case 'ios':
+    case 'android':
+      break;
+    default:
+      fail(`Expected \`platform\` to be one of ['ios', 'android']. Found '${options.platform}'.`);
+      return false;
+  }
+  return true;
+}
+
+const platformDefaults = {
+  ios: {
+    runnerAppPath: iosAppPath,
+    platformName: 'iOS',
+    platformVersion: '9.3',
+    deviceName: 'iPhone 6',
+    deviceOrientation: 'portrait',
+  },
+  android: {
+    runnerAppPath: androidAppPath,
+    platformName: 'Android',
+    platformVersion: '6.0',
+    deviceName: 'Android Emulator',
+    deviceOrientation: 'portrait',
+  },
+};
+
+// non-overridable capabilities
+const platformCaps = {
+  ios: {
+    automationName: 'xcuitest',
+  },
+  android: {
+    appPackage: 'com.HappoRunner',
+    appActivity: '.MainActivity',
+  },
+};
+
+module.exports = passedOptions => validatePassedOptions(passedOptions) && ({
+  name: 'react-native',
+  port: 5000, // websockets port
+  packagerCommand: 'node',
+  packageArgs: [
+    './node_modules/react-native/local-cli/cli.js',
+    'start',
+  ],
+  packagerCwd: process.cwd(),
+  packagerEnv: process.env,
+
+  appiumPath: './node_modules/.bin/appium',
```

Comment:
yeah i guess i just made it configurable and called that "good enough", but i think I should see if using `npm bin` will get me what i need or not, which may be a more robust way to do this and eliminate it as a config option